tests/components/yeelight/__init__.py

```diff
@@ -180,7 +180,14 @@ def _patch_discovery(no_device=False, capabilities=None):
 
     def _generate_fake_ssdp_listener(*args, **kwargs):
         return _patched_ssdp_listener(
-            None if no_device else capabilities or CAPABILITIES,
+            None
+            if no_device
+            else ssdp.SsdpServiceInfo(
+                ssdp_usn="mock_usn",
+                ssdp_st="mock_st",
+                upnp={},
+                ssdp_headers=capabilities or CAPABILITIES,
+            ),
```

Comment:
This should be unpacked into an `if` block since it spans multiple lines

Reply:
Done